### DISCUSSION TITLE
Director ".fusion-body..." for sites without Director Data

### DISCUSSION THREAD

**Discussion by jfreiman** [CodyBerenson/PGMA-Modernized#138] [Q&A] [answered]
I have an annoying issue that some videos which get matched (multiple sites) do not contain a Director and instead of leaving the Director Tab blank, it leaves a long string of code beginning with, ".fusion-body..." and continues for several lines.

below are screen captures of what I'm referring to

[https://1drv.ms/u/s!AqXiusoHW-XfxMBk3YvhylCymQ4mjw?e=9dG5AR](url)
[https://1drv.ms/u/s!AqXiusoHW-XfxMBk3YvhylCymQ4mjw?e=4jLoUX](url)

I'm hoping there is a simple solution that doesn't require me to manually delete the Plex Director tab manually.
Thank you in advance,
John

**Comment by JPH71** [comment, marked as answer]
Changes made - Cody will review changes.

**Reply by jfreiman** [reply]
thank you

**Comment by CodyBerenson** [comment]
@jfreiman 

When you say "multiple sites", can you confirm which they are?  Jason updated the GayWorld agent, but I wonder if you mean more sites than just that one.  Thanks.

**Comment by jfreiman** [comment]
I've edited each of the Plex video entries that had the code inserted in the Director field, so I don't have an easy way to filter them and check each video.
I do remember noticing it didn't just happen from one source, it happened on more than one -- I know that doesn't help.
I'll play around with some more videos this weekend and start a list of which sites/sources this occurs with. My assumption was that it happended to any video where a Director was absent -- but since this hasn't come up before now (I've been experiencing this issue for nearly a year?) it might not be as widespread as I imagined.

**Comment by CodyBerenson** [comment]
Hi @jfreiman ,

I did a search across my entire collections, and I found this issue for two site:  GayMovie and GayWorld.  Jason has coded the fix for GayMovie and is working on GayWorld.  If you come across any other sites, do let us know.

**Comment by CodyBerenson** [comment]
The GayMovie Agent has been updated.  See #141

**Comment by JPH71** [comment]
GayWorld - the site has been taken offline - we can take the agent out of
the repository....
Damn!

**Comment by jfreiman** [comment]
Thank you for all your effort and thoroughness – I truly appreciate your software!

PS. I’m guessing you have an extensive gay adult video collection, but if you want to check out possibly more videos as well as actual movies and Television programs please let me know and I’ll be more than happy to share my collections with you.

My Plex username is: jfreiman

***@***.******@***.***

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________

**Comment by jfreiman** [comment]
BTW. Before I discovered your software, I was literally inputting ALL the metadata by hand, copying and pasting from GEVI using both Foobar2000 and MP4 Video & Audio Tag Editor – thank you again,

John

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
